protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -748,10 +757,21 @@ message ListLedgerEntriesRequest {
     optional DateRange date_range = 3;
 }
 
+message ExportLedgerEntriesRequest {
+    string party_id = 3;
+    string asset_id = 4;
```

Comment:
Doesn't this need a `optional Pagination pagination = 6;` field to be able to use the cursor in the the response?